massa-hash/src/hash.rs

```diff
@@ -84,6 +133,16 @@ impl Hash {
     /// let hash = Hash::compute_from(&"hello world".as_bytes());
     /// let serialized: String = hash.to_bs58_check();
     /// ```
+    /// All massa addresses, whenever presented as user-readable texts,
```

Comment:
This is not true. Addresses are not only base58 there is prefix also in front of them. I also think that speaking of address in `massa-hash` could lead to confusion as Addresses doesn't exist in this crate.

We could speak only of hashes

Reply:
Fixed